packaging/debian/control

```diff
@@ -0,0 +1,29 @@
+Source: sscg
+Section: utils
+Priority: optional
+Build-Depends: debhelper-compat (= 12),
+ meson,
+ help2man,
+ libpath-utils-dev,
+ libpopt-dev,
+ libssl-dev,
+ libtalloc-dev,
+ pkgconf,
+Maintainer: Stephen Gallagher <sgallagh@redhat.com>
+Uploaders: Martin Pitt <mpitt@debian.org>
```

Comment:
For upstream this doesn't matter, but for downstream you need to decide whether you want to be the official maintainer -- in other words, get bug reports and such. Of course I'm happy to do the actual uploads and version updates (I don't think there'll be that many), that's what "Uploaders:"  means (essentially, "co-maintainers")

Reply:
I don't mind being the official maintainer.